dotnet/src/Azure.Iot.Operations.Protocol/RPC/CommandResponseMetadata.cs

```diff
@@ -47,11 +47,12 @@ public class CommandResponseMetadata
         public Dictionary<string, string> UserData { get; set; } = new();
 
         /// <summary>
-        /// The CloudEvent metadata attached to the response.
+        /// CloudEvent metadata for this RPC response. The Type will be automatically set to "ms.aio.rpc.response"
```

Comment:
is this struct for the sending and receiving side? If so, might want to have some clarification that this is only for the executor side of things. On the receiving side, the Type would be set from the user properties and the DataContentType would be set from the publish properties as well